CMakeLists.txt

```diff
@@ -67,18 +66,21 @@ if(WINDOWS_STORE)
         "Source/UWP/arcana/threading/task_conversions.h"
         "Source/UWP/arcana/threading/task_schedulers.h"
         "Source/UWP/arcana/threading/task_schedulers.cpp")
-endif()
 
-add_library(arcana ${SOURCES})
+    add_library(arcana ${SOURCES})
 
-target_include_directories(arcana PUBLIC "Source/Submodules/GSL/include")
-target_include_directories(arcana PUBLIC "Source/Shared")
+    target_include_directories(arcana PUBLIC "Source/Submodules/GSL/include")
+    target_include_directories(arcana PUBLIC "Source/Shared")
```

Comment:
Can these lines be moved out of the conditional blocks since they are the same (e.g. down to line 87)? Or is there an ordering issue here?

Reply:
They're not quite the same. The include folders here are added as `PUBLIC`, meaning they expose themselves to both the library's own .cpp files and to consuming libraries. They're `INTERFACE` below, though, because in CMake header-only libraries aren't allowed to have `PUBLIC` includes (presumably because they don't have any .cpp files).